cypress/locators/duelling-picklist/index.js

```diff
@@ -1,17 +1,19 @@
 import {
   DUELLING_PICKLIST_COMPONENT, PICKLIST, PICKLIST_ITEMS, PICKLIST_LEFT_LABEL,
-  PICKLIST_RIGHT_LABEL, ADD_ELEMENT, REMOVE_ELEMENT, PICKLIST_DIVIDER,
+  PICKLIST_RIGHT_LABEL, ADD_ELEMENT, REMOVE_ELEMENT, PICKLIST_DIVIDER, PICKLIST_ID,
```

Comment:
Should we rename it `PICKLIST_ID` -> `DUELLING_PICKLIST_ID`

Reply:
All locators have just `picklist` name, so I think it's no needed